website/src/pages/contact/sales.tsx

```diff
@@ -24,57 +24,56 @@ export default class Trial extends React.Component<any, any> {
-                                </li>
-                            </ul>
-                            <div className="panel__help">
-                                <strong>Need help?</strong>
-                                <p>
-                                    Contact us at <a href="https://twitter.com/srcgraph">@srcgraph</a> or{' '}
-                                    <a href="mailto:support@sourcegraph.com">support@sourcegraph.com</a>, or file issues
-                                    on our{' '}
-                                    <a href="https://github.com/sourcegraph/issues/issues">public issue tracker</a>.
-                                </p>
+                    <section className="d-flex justify-content-around flex-column flex-lg-row sales">
+                        <div className="panel-area mr-lg-5 mt-0">
+                            <div className="d-flex flex-column panel">
+                                <h3>Your team will love Sourcegraph Enterprise</h3>
+                                <p>The best way to search and browse all of your organization's code, with:</p>
+                                <ul className="list pl0">
+                                    <li className="mb-2">
+                                        <div className="">HA cluster support</div>
+                                    </li>
+                                    <li className="mb-2">Cross-repository code intelligence</li>
+                                    <li className="mb-2">Advanced logging</li>
+                                    <li className="mb-2">External database support</li>
+                                    <li className="mb-2">Premium support</li>
+                                    <li>Private Sourcegraph extension registry</li>
+                                </ul>
+                                <div className="panel__help">
+                                    <strong>Need help?</strong>
+                                    <p>
+                                        Contact us at <a href="https://twitter.com/srcgraph">@srcgraph</a> or{' '}
+                                        <a href="mailto:support@sourcegraph.com">support@sourcegraph.com</a>, or file
+                                        issues on our{' '}
+                                        <a href="https://github.com/sourcegraph/issues/issues">public issue tracker</a>.
+                                    </p>
+                                </div>
                             </div>
                         </div>
-                    </div>
-                </section>
+                        <div className="form-area">
+                            <div>
+                                <h1>Register for your free trial</h1>
+                            </div>
+                            <div className="form">
+                                <div id="hubspotTrialForm" className="d-flex justify-center" />
```

Comment:
I can't comment on the form itself, but the mixing of verbs over here, with "Register" in the header, and "Get Started" in the button, seems a bit confusing. What do you think?

Reply:
I didn't find it confusing (I've seen other forms use the same verbs) - You put in your information which is registering and then you click get started. If you have any suggestions that you think sound better happy to that instead.

Also, I removed submit because a user has next steps immediately whereas before they only had a confirmation they would get a response.